connector/routingconnector/router.go

```diff
@@ -29,8 +30,9 @@ type router[C any] struct {
 	logger *zap.Logger
 	parser ottl.Parser[ottlresource.TransformContext]
 
-	table  []RoutingTableItem
-	routes map[string]routingItem[C]
+	table      []RoutingTableItem
+	routes     map[string]routingItem[C]
+	routeSlice []routingItem[C]
```

Comment:
I'm not very familiar with this connector, but in what scenario do we end up with multiple `table` items? (I may or may not have some functionality suggestions depending on the answer.)

Reply:
Hi, @crobert-1 . Do you mean why the table is an array? Or why do we need `table`, `routes`, and `routeSlice` to store statements? If it's the former, it's because the `table` is used to contain multiple statements. If it's the latter, `routeSlice` is used to sort the statements, while `routes` is not used.
If I have misunderstood, please feel free to correct me.